hathor/p2p/peer_storage.py

```diff
@@ -50,3 +54,11 @@ def remove(self, peer: Peer) -> None:
         assert peer.id is not None
         if peer.id in self:
             del self[peer.id]
+
+
+class PeerStorage(BasePeerStorage[PublicPeer]):
+    pass
+
+
+class ReceivedPeerStorage(BasePeerStorage[Peer]):
```

Comment:
The acceptance criteria does not mention this new storage. Can you add some wording on what it is supposed to do?

Reply:
I'll add it.

Reply:
Thanks!